packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -58,6 +58,11 @@
     flex: 1;
     margin-bottom: 0;
     margin-top: var(--terra-demographics-banner-person-details-margin-top);
+
+    // IE 11
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: 0;
```

Comment:
margin-top is already set zero in both fusion and low-light theme. So do we still need this rule for IE 11 ?

Reply:
in that case instead of this checks for IE11 we can directly update line no. 60 `margin-top: var(--terra-demographics-banner-person-details-margin-top, 0);`